moped-editor/src/views/staff/StaffForm.js

```diff
@@ -175,6 +172,9 @@ const StaffForm = ({ editFormData = null, userCognitoId }) => {
       delete data.password;
     }
 
+    // Reactivation needs this value to pass with programmatic submission in handleActivateConfirm
```

Comment:
Not sure if this is the best place for this. When a deactivated user was activated previously, the submission payload took advantage of the hidden field that I removed further down in the diff. That hidden field really isn't needed since `is_deleted` initializes as `false` for new users (it used to need `status_id = 1` in the payload).

Also, I'm making a backlog issue to split out the create and edit logic into two components. Create and edit actions (like deactivate/activate) are pretty tangled here. I'm all over the blame for this code so I'm happy to improve this when there is priority for this. 🖐️ 😄

Reply:
👍👍👍